packages/psss/src/utils/auth.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { AccessPolicy } from '@tupaia/access-policy';
+
+export const checkIsAuthorisedForMultiCountry = user => {
+  const activeEntity = getActiveEntityByUser(user);
+  return Array.isArray(activeEntity);
+};
+
+export const checkIsAuthorisedForCountry = (user, match) => {
+  if (checkIsAuthorisedForMultiCountry(user)) {
+    return true;
+  }
+
+  return getActiveEntityByUser(user) === match.params.countryCode;
+};
+
+export const getActiveEntityByUser = user => {
```

Comment:
```suggestion
export const getPermittedEntitiesForUser = user => {
```
What do you think about a rename everywhere from `activeEntity` to `permittedEntities` (an array, as per previous comment)